hermes/brokers/hopskotch.py

```diff
@@ -765,11 +789,11 @@ def get_user_topic_permissions(username, credential_name, user_api_token,
         # add the publicly_readable topics to the list of readable topics fro the user
         public_topic_names = [topic['name'] for topic in get_topics(user_api_token, publicly_readable_only=True)]
         user_topic_permissions['read'] = list(set(user_topic_permissions['read'] + public_topic_names))
-
-    for group in exclude_groups:
-        # filter topics owned by groups in the exclude_groups list of group names
-        user_topic_permissions['write'] = [topic for topic in user_topic_permissions['write'] if not topic.startswith(group)]
-        user_topic_permissions['read'] = [topic for topic in user_topic_permissions['read'] if not topic.startswith(group)]
+    if exclude_groups:
+        for group in exclude_groups:
```

Comment:
is this for style? with the default being [], I view `for x in []:` as a no-op. or, is that too obsure?